Formula/php.rb

```diff
@@ -0,0 +1,271 @@
+    end
+
+    args << "--enable-maintainer-zts" if build.with? "thread-safety"
+
+    system "./buildconf", "--force"
+    system "./configure", *args
+
+    inreplace "Makefile",
+      /^INSTALL_IT = \$\(mkinstalldirs\) '([^']+)' (.+) LIBEXECDIR=([^\s]+) (.+)$/,
+      "INSTALL_IT = $(mkinstalldirs) '#{libexec}/apache2' \\2 LIBEXECDIR='#{libexec}/apache2' \\4"
+
+    # https://github.com/phpbrew/phpbrew/commit/18ef766d0e013ee87ac7d86e338ebec89fbeb445
+    # Unsure if this is still needed
+    inreplace "Makefile" do |s|
+      s.change_make_var! "EXTRA_LIBS", "\\1 -lstdc++"
+    end
+
+    system "make"
+    ENV.deparallelize
+    system "make", "install"
+
+    ln_s "phar.phar", bin+"phar", :force => true
+
+    # Install new php.ini unless one exists
+    config_path.install "./php.ini-development" => "php.ini" unless File.exist? config_path+"php.ini"
+
+    chmod 0755, "sapi/fpm/init.d.php-fpm"
+    sbin.install "sapi/fpm/init.d.php-fpm" => "php#{version.to_s[0..2].delete(".")}-fpm"
+
+    if File.exist?("sapi/cgi/fpm/php-fpm")
+      chmod 0755, "sapi/cgi/fpm/php-fpm"
+      sbin.install "sapi/cgi/fpm/php-fpm" => "php#{version.to_s[0..2].delete(".")}-fpm"
+    end
+
+    if !File.exist?(config_path+"php-fpm.d/www.conf") && File.exist?(config_path+"php-fpm.d/www.conf.default")
+      mv(config_path+"php-fpm.d/www.conf.default", config_path+"php-fpm.d/www.conf")
+    end
+
+    unless File.exist?(config_path+"php-fpm.conf")
+      if File.exist?("sapi/fpm/php-fpm.conf")
+        config_path.install "sapi/fpm/php-fpm.conf"
+      end
+
+      if File.exist?("sapi/cgi/fpm/php-fpm.conf")
+        config_path.install "sapi/cgi/fpm/php-fpm.conf"
+      end
+
+      inreplace config_path+"php-fpm.conf" do |s|
+        s.sub!(/^;?daemonize\s*=.+$/, "daemonize = no")
+      end
+    end
+  end
+
+  def caveats
+    s = []
+
+    s << <<-EOS.undent
+      To enable PHP in Apache add the following to httpd.conf and restart Apache:
+          LoadModule php7_module #{HOMEBREW_PREFIX}/opt/php#{version.to_s[0..2].delete(".")}/libexec/apache2/libphp7.so
+
+          <FilesMatch \.php$>
+              SetHandler application/x-httpd-php
+          </FilesMatch>
+
+      Finally, check DirectoryIndex includes index.php
+          DirectoryIndex index.php index.html
+    EOS
+
+    s << <<-EOS.undent
+      The php.ini file can be found in:
+          #{etc}/php/#{version.to_s.split(".")[0..1].join(".")}/php.ini
+    EOS
+
+    s.join "\n"
+  end
+
+  def post_install
+    (prefix/"var/log").mkpath
+    touch prefix/"var/log/php-fpm.log"
+
+    chmod 0755, lib/"php/.channels"
+    chmod 0755, lib/"php/.channels/.alias"
+    chmod 0644, (Dir.glob(lib/"php/.channels/**/*", File::FNM_DOTMATCH).reject { |a| a =~ /\/\.{1,2}$/ || File.directory?(a)})
+
+    %w[
+      php/.depdblock
+      php/.filemap
+      php/.depdb
+      php/.lock
+    ].each do |f|
+      chmod 0644, lib/f
+    end
+
+    php_ini = "#{etc}/php/#{version.to_s.split(".")[0..1].join(".")}"
```

Comment:
This is the directory (missing `php.ini` file), which is what is causing pear/pecl to fail to write to the ini file during installation.